compute/README.md

```diff
@@ -0,0 +1,55 @@
+---
+page_type: sample
+languages:
+- golang
+products:
+- azure
+description: "These code samples will show you how to manage Compute using Azure SDK for Golang."
+urlFragment: compute
+---
+
+# Getting started - Managing Compute using Azure Golang SDK
+
+These code samples will show you how to manage Compute using Azure SDK for Golang.
+
+## Features
+
+This project framework provides examples for the following services:
+
+### Compute
+* Using the Azure SDK for Golang - Compute Management Library [compute/armcompute](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/compute/armcompute) for the [Azure Compute API](https://docs.microsoft.com/en-us/rest/api/compute/)
+
+### Prerequisites
+* an [Azure subscription](https://azure.microsoft.com)
+* Go 1.13 or above
+
+### Quickstart
+
+1. Clone the repository.
+
+    ```
+    git clone https://github.com/Azure-Samples/azure-sdk-for-go-samples.git
+    ```
+2. Set the environment variable.
+
+   ```
+   #Linux
+   export AZURE_SUBSCRIPTION_ID=<your Azure subscription id> 
+   # If no value is set, the created resource will be deleted by default.
+   export KEEP_RESOURCE=
+   
+   #PowerShell
+   $env:AZURE_SUBSCRIPTION_ID=<your Azure subscription id> 
+   export KEEP_RESOURCE=
+   ```
```

Comment:
We need to move the information of `linux` and `windows` out of the code block. Also to make things clear we should use `Linux` vs `Windows` or `bash` vs `powershell` instead of mixing these up.

Also we should `export KEEP_RESOURCES=1` (anything other than empty to keep the resources.